votor/src/certificate_pool/vote_certificate.rs

```diff
@@ -33,73 +31,92 @@ pub enum CertificateError {
     InvalidVoteType,
 }
 
-//TODO(wen): Maybe we can merge all the below functions into CertificateMessage.
+/// A builder for creating a `CertificateMessage` by efficiently aggregating BLS signatures.
 #[derive(Clone)]
-pub struct VoteCertificate(CertificateMessage);
+pub struct VoteCertificateBuilder {
```

Comment:
Nice, let's change the file name as well.

Reply:
Done!